camelot/view/validator.py

```diff
@@ -45,6 +45,21 @@
 
 data_validity = collections.namedtuple('data_validity', ['valid', 'value', 'formatted_value', 'error_msg', 'info'])
 
+@dataclass
+class ValidatorState(DataclassSerializable):
+
+    value: str = None
+    formatted_value: str = None
+    valid: bool = True
```

Comment:
Misschien kan deze bool vervangen worden door een QtGui.QValidator.State? Dat maakt de conversie in de validator zelf eenvoudiger en laat ook meer dingen toe in de toekomst.